src/course/config.json

```diff
@@ -12,7 +12,13 @@
         "_isEnabledOnTouchDevices": false,
         "_shouldSupportLegacyBrowsers": true,
         "_isTextProcessorEnabled": true,
-        "_isSkipNavigationEnabled": true
+        "_isSkipNavigationEnabled": true,
+        "_ariaLevels": {
```

Comment:
we need a way of overriding this for each article, block, component etc. either default / specified on the model. this is very much dependent on the course content.